setup.py

```diff
@@ -335,7 +335,7 @@ def get_config_var(name):
 # ==========
 setup(
     name='Cartopy',
-    version='0.14.4.dev0',
+    version='0.15.0',
```

Comment:
@lbdreyer What if the `version_switch.js` was to read a json file in order to initialise the `all_versions` ?

The json file could be generated as part of `make html` or thought some other mechanism?

Perhaps not part of this PR, but I'll raise an issue to that this isn't forgotten.